qhana_plugin_runner/db/models/tasks.py

```diff
@@ -162,10 +157,12 @@ class ProcessingTask:
 
     current_step: int = field(default=-1, metadata={"sa": Column(sql.Integer())})
 
-    progress_value: int = field(default=0, metadata={"sa": Column(sql.Integer())})
+    progress_value: int = field(
+        default=None, metadata={"sa": Column(sql.Integer(), nullable=True)}
```

Comment:
Progress values should use floating point numbers. They are accurate enough for some progress estimate and allow for fractional values which can be good for time estimates.